### DISCUSSION TITLE
如何设置初始地图zoom

### DISCUSSION THREAD

**Discussion by 1183011789** [zouyaoji/vue-cesium#146] [General]
![1642749798(1)](https://user-images.githubusercontent.com/30824602/150484128-9a981482-2d94-4725-b60a-25b9fc9ffd38.jpg)

**Comment by zouyaoji** [comment]
@1183011789 
There is no zoom concept in cesium like a two-dimensional map, so there is no way to set zoom.
cesium里面不像二维地图那样有zoom的概念，所以是没办法设置zoom的。

The camera parameters should be used to initialize the map field of view in cesium.
cesium里面应该用相机参数来初始化地图视野。

That is, the camera property of the vc-viewer component.
即vc-viewer组件的camera属性。

Or the vc-navigation component also has related properties.
或者vc-navigation组件也有相关属性。
